cfme/tests/control/test_bugs.py

```diff
@@ -515,3 +556,59 @@ def test_edit_action_buttons(action_for_testing):
     # click the cancel button and navigate to "Details" so that the action can be deleted
     view.cancel_button.click()
     navigate_to(action_for_testing, "Details")
+
+
+@pytest.mark.meta(blockers=[BZ(1717483)], automates=[1711352])
+def test_policy_condition_multiple_ors(
+        appliance,
+        virtualcenter_provider,
+        vm_compliance_policy_profile
+):
+    """
+    Tests to make sure that policy conditions with multiple or statements work properly
+
+    Bugzilla:
+        1711352
+
+    Polarion:
+        assignee: jdupuy
+        caseimportance: low
+        casecomponent: Control
+        initialEstimate: 1/12h
+    """
+    collection = appliance.provider_based_collection(virtualcenter_provider)
+    all_vms = collection.all()
+    all_vm_names = [vm.name for vm in all_vms]
+
+    # we need to select out cu-24x7
+    vm_name = virtualcenter_provider.data["cap_and_util"]["capandu_vm"]
+    # check that it exists on provider
+    if not virtualcenter_provider.mgmt.does_vm_exist(vm_name):
+        pytest.skip("No capandu_vm available on virtualcenter_provider of name {}".format(vm_name))
+
+    vms = [all_vms.pop(all_vm_names.index(vm_name))]
+
+    # add some other vms to the list to run the policy simulation against
+    # we use this ugly logic in case there are not that many vms on the provider
+    if len(all_vms) > 3:
```

Comment:
slightly less messy:

if len(all_vms) == 1:
    vms.extend(all_vms[0])
else:
    vms.extend(all_vms[0:len(all_vms)%5]
```

Reply:
Even better solution found via discussion, since this `%` could result in zeros, `min(randint(1, len(all_vms)), 5)`